Rocket.Chat/Controllers/Chat/ChannelActionsViewController.swift

```diff
@@ -45,6 +46,10 @@ class ChannelActionsViewController: BaseViewController {
                         title: localized("chat.info.item.description"),
                         descriptionText: hasDescription ? subscription.roomDescription : localized("chat.info.item.no_description")
                     ),
+                    ChannelInfoDescriptionCellData(
+                        title: "Announcement",
+                        descriptionText: hasAnnouncement ? subscription.roomAnnouncement : "No announcement"
```

Comment:
Let's localize the string `"No announcement"` please!

Reply:
Yes, I overlooked this! Am I supposed to add a string to each of the files and Google translate each language? Or do I just add the English version only?

Reply:
@blewjy Just English is fine, you can just input the English translation in all other languages and you can use **Sem anúncio** for `pt-BR` 💯

Reply:
@rafaelks Done!